### PR TITLE
Various: Grammar and wording fixes (codefree)

### PR DESCRIPTION
**Pull request by Intelrunner**
Issue: Wording and grammar errors

Impact: Causes reduced clarity in overall meaning and messaging

Fix: Correct within file.

Outcome:
Cleared up some language and punctuation in file comments/non-code areas.

### CODE REVIEW

**Comment by celestehorgan** [comment]
Hi @Intelrunner!

We need you to sign the contributor license agreement first :) You can do so here https://identity.linuxfoundation.org/projects/cncf

**Comment by Intelrunner** [comment]
I signed the CLA.

**Comment by Intelrunner** [comment]
> Hi @Intelrunner!
> 
> We need you to sign the contributor license agreement first :) You can do so here https://identity.linuxfoundation.org/projects/cncf

Thank you so much. I appreciate the assistance. First commit.

**Review by celestehorgan — COMMENTED** [review]
Hi @Intelrunner,

Thank you for submitting this change, but upon further review either:

- The original grammar is correct, or
- The suggested changes are grammatically equivalent to the original

As such, I cannot merge this change for you. 

However! We appreciate that you took the time to do grammar edits to the documentation! There are _lots_ of pages which need this kind of work. May I suggest editing [the Service Topology page](https://kubernetes.io/docs/concepts/services-networking/service-topology/)? This would page would be a great candidate for the kind of work you're looking to do :)

File: content/en/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <div class="row">
       <div class="col-md-9">
         <h2>Kubernetes Basics</h2>
-        <p>This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
+        <p>This tutorial provides a walkthrough of Kubernetes (or K8s) cluster orchestration system basics. Each module contains some background information on major Kubernetes features and concepts, and includes an interactive online tutorial. These interactive tutorials let you manage a simple cluster and its containerized applications for yourself.</p>
```
Comment (celestehorgan) [inline content/en/docs/tutorials/kubernetes-basics/_index.html:27]: To clarify my earlier comments: The original grammar here, `This tutorial provides a walkthrough of the basics of the Kubernetes cluster orchestration system.` is more correct than this.

File: content/en/docs/tutorials/kubernetes-basics/_index.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h2>Kubernetes Basics</h2>
...
+        <p>With modern web services, users expect applications to be available 24/7 and developers expect to deploy new versions of those applications several times a day. Containerization helps package software to serve these goals; enabling applications to be released and updated in an easy and fast way without downtime. Kubernetes helps you be sure the containerized applications run where and when you want, and helps the containers find the resources and tools they need to work. Kubernetes is a production-ready, open source platform designed with Google's accumulated experience in container orchestration, combined with best-of-breed ideas from the community.</p>
```
Comment (celestehorgan) [inline content/en/docs/tutorials/kubernetes-basics/_index.html:44]: And in this chunk, your proposed replacement is functionally equivalent to the original.

**Pull request closed without merging**